src/Protocol/ActivityPub/Processor.php

```diff
@@ -81,6 +82,45 @@ private static function replaceEmojis($body, array $emojis)
 		return $body;
 	}
 
+	/**
+	 * Store attached media files in the post-media table
+	 *
+	 * @param int $uriid
+	 * @param array $attachment
+	 * @return void
+	 */
+	private static function storeAttachment(int $uriid, array $attachment)
```

Comment:
Please rename as `storeAttachmentAsMedia`
```suggestion
	private static function storeAttachmentAsMedia(int $uriid, array $attachment)
```

Reply:
Done.

Reply:
I did not see this change in the recent commit.

Reply:
Sorry, forgot this. Just added.